PrivateClinicsNetWebApi/ServicesConfigurator.cs

```diff
@@ -0,0 +1,126 @@
+﻿using Microsoft.AspNetCore.Authentication.JwtBearer;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.EntityFrameworkCore;
+using PrivateClinicsWebNet.DataAccess;
+using PrivateClinicsWebNet.BusinessLogic.Repositories;
+using System.Text;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.OpenApi.Models;
+using System.Security.Cryptography.Xml;
+using PrivateClinicsWebNet.Application.Services;
+using PrivateClinicsWebNet.DataAccess.Services;
+using PrivateClinicsWebNet.DataAccess.Abstractions;
+
+namespace PrivateClinicsNetWebApi
+{
+    public class ServicesConfigurator
```

Comment:
Good job, but let's use common approach: We create different `ServiceCollectionExtensions` classes and present extension methods. Like `AddPostgres()`, `AddJwtAuth`, `AddBusinessLogic` etc